recipe/meta.yaml

```diff
@@ -23,7 +23,7 @@ requirements:
     - azure-core >=1.5.0
     - azure-datalake-store >=0.0.46,<0.1
     - azure-identity
-    - azure-storage-blob >=12.5.0,<12.7.0
+    - azure-storage-blob >=12.5.0,<12.9.0
```

Comment:
In upstream there isn't an upper bound: https://github.com/dask/adlfs/blob/e2a3807b8748a48eec366730a6d6ed60e4192cf4/setup.py#L35. Just match that?

Reply:
I was being conservative as I figure it's easier to put in a PR to bump the pin than it is to patch the repodata to add a pin after the fact.

Happy to make the change to follow upstream though...